### DISCUSSION TITLE
zh/guide/advanced/backup

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#357] [Announcements]
# zh/guide/advanced/backup

方法1-内置 使用AList后台的内置 备份/恢复选项 1. 备份：进行AList数据进行备份（==不包含索引==） 2. 恢复：恢复之前备份文件 3. 覆盖：恢复时选择覆盖会将用户信息进行覆盖 4. 加密：填写加密密码导出时会将信息进行加密，导入时也需要先输入加密密码再点击恢复^3.40.0_新增^ 方法2-sqlite3 直接将AList目录下的 ...

https://alist.nn.ci/zh/guide/advanced/backup.html

**Comment by yyyytawa** [comment]
版本号是不是写错了 3.40.0 现在·最新也才3.35
